test/_unit/custom-validations.spec.js

```diff
@@ -0,0 +1,18 @@
+const countries = require('hof').utils.countries().concat([{ value: 'Unknown', label: 'Unknown' }]);
+const isInCountriesList = value => countries.some(country => country.value === value);
```

Comment:
This test is ok in that it shows some tdd but the function under test is internal making the test purely self referential and therefore redundant. This is a minor thing but it adds to the overall maintenance burden of the project and may be better placed in support documentation. This is only my opinion, no further action required.

Reply:
I agree...I was concerned that exporting the function for test from the acrs/fields/index.js would play havoc with HOF build but on examination after update it seems fine.

Have updated to pull the function from the place it actually lives rather than duplicating it.